src/dbusextendedabstractinterface.h

```diff
@@ -0,0 +1,99 @@
+// -*- c++ -*-
+
+/*!
+ *
+ * Copyright (C) 2015 Jolla Ltd.
+ *
+ * Contact: Valerio Valerio <valerio.valerio@jolla.com>
+ * Author: Andres Gomez <andres.gomez@jolla.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+
+#ifndef DBUSEXTENDEDABSTRACTINTERFACE_H
+#define DBUSEXTENDEDABSTRACTINTERFACE_H
+
+#include <DBusExtended>
+
+#include <QDBusAbstractInterface>
+#include <QDBusError>
+
+class QDBusPendingCallWatcher;
+class DBusExtendedPendingCallWatcher;
+
+class QT_DBUS_EXTENDED_EXPORT DBusExtendedAbstractInterface: public QDBusAbstractInterface
+{
+    Q_OBJECT
+
+public:
+    virtual ~DBusExtendedAbstractInterface();
+
+    Q_PROPERTY(bool sync READ sync WRITE setSync)
+    inline bool sync()
+    { return m_sync; }
```

Comment:
Nitpick, in Qt code normally such inline function are all in the same line if not too big